declarations/TikTok.json

```
@@ -41,7 +41,9 @@
     },
     "Trackers Policy": {
       "fetch": "https://www.tiktok.com/legal/tiktok-website-cookies-policy?lang=en",
```

This document seems to have moved to the following url : `https://www.tiktok.com/legal/page/global/tiktok-website-cookies-policy/en`

```suggestion
      "fetch": "https://www.tiktok.com/legal/page/global/tiktok-website-cookies-policy/en",
```